crates/sui-json/src/tests.rs

```diff
@@ -334,7 +335,7 @@ fn test_basic_args_linter_pure_args_good() {
         (
             Value::from(good_hex_val),
             MoveTypeLayout::Vector(Box::new(MoveTypeLayout::U8)),
-            bcs::to_bytes(&hex::decode(good_hex_val.trim_start_matches(HEX_PREFIX)).unwrap())
+            bcs::to_bytes(&Hex::decode(good_hex_val.trim_start_matches(HEX_PREFIX)).unwrap())
```

Comment:
which of the two functions do we need here?

Reply:
so Hex::decode tries to strip 0x and then hex::decode. effectively the same